build.cmd

```diff
@@ -1,41 +1,4 @@
-@if not defined _echo @echo off
-setlocal
-
-if /I [%1] == [-?] goto Usage
-
-if [%1] == [] goto Build
-
-set _rootDirectory=%CD%\
-if EXIST %_rootDirectory%%1 goto :BuildDirectory
-set _rootDirectory=%~dp0src\
-if EXIST %_rootDirectory%%1 goto :BuildDirectory
-
-:Build
-call %~dp0build-native.cmd %*
-if NOT [%ERRORLEVEL%]==[0] exit /b 1
-call %~dp0build-managed.cmd -BuildPackages=true %*
+@echo off
+powershell -ExecutionPolicy ByPass -NoProfile %~dp0eng\Build.ps1 -restore -build -warnaserror:0 -nodereuse:0 %*
+echo Build.cmd ErrorLevel=%ERRORLEVEL%
```

Comment:
NIT: I might be wrong, but I believe that this line will make your next line to always exit with the ERRORLEVEL of 0. I think it only stores the output of the last command that ran, so the echo command will return this back to a 0 so we will always exit /b 0. We could either not print this out, or save the error returned from powershell into a variable that we can then both print, and exit with.

Reply:
`echo` is an intrinsic, not a command, so it doesn't update ERRORLEVEL